kie-internal/src/main/java/org/kie/internal/security/KiePolicyHelper.java

```diff
@@ -60,7 +60,6 @@ public class KiePolicyHelper {
         } catch (Exception e) {
             ctx = null;
             log.error("Error loading and installing KIE security policy.", e);
-            e.printStackTrace();
```

Comment:
the exception is already captured in SLF4J logging appropriately in the line above